src/benchmarks/micro/runtime/System.Reflection/Attributes.cs

```diff
@@ -67,11 +67,13 @@ public class MyAttribute : Attribute
         [Benchmark(Description = "GetCustomAttributes - Method Base: Hit (inherit)")]
         public object[] GetCustomAttributesMethodBaseHitInherit() => AttributedBase.GetCustomAttributes(typeof(MyAttribute), true);
         [Benchmark(Description = "GetCustomAttributes - Method Base: Miss (inherit)")]
+[MemoryRandomization]
```

Comment:
Double check formatting, I am not sure of correct formatting here.

Reply:
Same below

Reply:
I also added additional line breaks here for readability.